src/components/LoginForm/LoginForm.component.test.tsx

```diff
@@ -34,17 +34,17 @@ describe('LoginForm component', () => {
     expect(loginButton).toBeInTheDocument();
   });
 
-  it('handles submit function with provided data', async () => {
+  it('handles submit function with provided data', () => {
     render(<LoginForm onClose={onCloseMock} />);
 
-    const inputEmail = await screen.getByPlaceholderText('Email');
-    const inputPassword = await screen.getByPlaceholderText('Password');
-    const loginButton = await screen.getByRole('button', { name: 'Log In' });
+    const inputEmail = screen.getByPlaceholderText('Email');
+    const inputPassword = screen.getByPlaceholderText('Password');
+    const loginButton = screen.getByRole('button', { name: 'Log In' });
 
     fireEvent.change(inputEmail, { target: { value: 'test@test.com' } });
```

Comment:
**Recommendation**

We can use [user-events ](https://testing-library.com/docs/ecosystem-user-event/) library for the browser interactions such as click, type, hover, etc.

Instead of calling fireEvent.change we simply use userEvent.type like:

`userEvent.type(input, 'password')`

Reply:
Recommendations applied in LoginForm and SignupForm components! :)